molecule/default/prepare.yml

```diff
@@ -8,6 +7,14 @@
   pre_tasks:
     - name: Install python
       raw: apt update; apt install -y python3
+    - name: Install software-properties-common
+      raw: apt update; apt install -y software-properties-common
+    - name: Add deadsnakes PPA
+      raw: add-apt-repository ppa:deadsnakes/ppa; apt update
+    - name: Install python3.7
+      raw: apt install -y python3.7
+    - name: Gathering Facts
+      setup:
```

Comment:
I added this because the container needs python3.7 to gather facts, otherwise, I get two errors in the same message "SyntaxError: future feature annotations is not defined , "msg": "The following modules failed to execute: ansible.legacy.setup".